charts/airflow-operator/templates/manager/controller-manager-metrics-service.yaml

```diff
@@ -13,10 +13,11 @@ metadata:
   name: {{ .Release.Name }}-aocm-metrics-service
   namespace: '{{ .Release.Namespace }}'
 spec:
+  type: ClusterIP
   ports:
   - name: https
     port: {{ .Values.ports.managerContainerPort }}
-    targetPort: https
+    targetPort: {{ ternary "https" .Values.ports.managerUpstreamPort  .Values.manager.metrics.useSecuredEndpoint }}
```

Comment:
@pgvishnuram Do we have a container port named https? Can you point me to its remote URL?

Reply:
when useSecuredEndpoint is enabled we expose container port name as https its in helpers.tpl under airflow-operator